### DISCUSSION TITLE
可以不用gpu运行吗？

### DISCUSSION THREAD

**Discussion by xanxus514** [RL-VIG/LibFewShot#75] [Q&A]
我的笔记本没有显卡，不靠gpu只用cpu运行项目时，出现了下面的错误，希望作者能帮忙看一下
![image](https://user-images.githubusercontent.com/19319907/228752075-7cc646c2-457c-4c8e-82c0-57d34b18ed92.png)
我把device.yaml，test_install.yaml里的device_ids改成-1了，运行的代码是第一个测试是否安装成功的，如下。

![image](https://user-images.githubusercontent.com/19319907/228752885-9c7114d9-ee6b-45eb-8b28-8025c0533c83.png)

**Comment by wZuck** [comment]
+ 当前的代码版本已经不再考虑支持CPU，因为对训练测试来说开销都太大，还是建议寻求GPU资源，例如免费的[Google Colab](https://colab.research.google.com)。
+ 我们的文档有一段时间没更新了，跟着做会有点问题。可以直接尝试使用config/proto.yaml试一试。我们会在这段时间陆续更新一下文档内容。
